python/src/opendp/context.py

```diff
@@ -697,7 +697,6 @@ def __init__(self, f, *args, **kwargs):
 
     def __call__(self, v):
         """Returns the transformation or measurement with the given parameter."""
-        # TODO: Can we exercise this?
         return self.partial(v) # pragma: no cover
```

Comment:
```suggestion
        return self.partial(v)
```
Do you mean to include this as tested code now? The TODO isn't very useful, so no complaints about removing it: we can always find the `pragma`.